pallets/gear-program/src/migrations/allocations.rs

```diff
@@ -58,21 +62,28 @@ impl<T: Config> OnRuntimeUpgrade for MigrateAllocations<T> {
             let update_to = StorageVersion::new(MIGRATE_TO_VERSION);
             log::info!("🚚 Running migration from {onchain:?} to {update_to:?}, current storage version is {current:?}.");
 
-            ProgramStorage::<T>::translate(|_, program: v5::Program<BlockNumberFor<T>>| {
+            ProgramStorage::<T>::translate(|id, program: v5::Program<BlockNumberFor<T>>| {
```

Comment:
- shouldn't you bump storage version and applicable constants?
- could you please rename this migration? (note: its about to be removed since already took place on both networks, so patching this one looks ok - just want to make things more clear)

Reply:
You can see now

Reply:
suppose it's outdated now